model/RdsResultStorage.php

```diff
@@ -474,15 +474,28 @@ public function getAllDeliveryIds()
     public function getResultByDelivery($delivery, $options = array())
     {
         $returnValue = array();
+        $ids_sql='';
+        if(isset($options['start_time'])){
+            $ids_sql = 'SELECT '.DeliveryMonitoringService::DELIVERY_EXECUTION_ID.' from delivery_monitoring 
+            WHERE start_time>='.$options['start_time']. 'AND start_time<='.$options['end_time'];
```

Comment:
we can use either start or end time
add please:
```
if (array_key_exists('start_time', $options) or array_key_exists('end_time', $options))
```
then for the conditions:
```
if (array_key_exists('start_time', $options)) {$where[]='start_time>='.$options}
// the same for the end time
$where = implode(' AND ', $where)
```

Reply:
Als you can't use DeliveryMonitoring data for this data collector, we don't have such dependency between extensions.
Please, use ServiceProxy instead